app/helpers/loans_helper.rb

```diff
@@ -2,4 +2,29 @@ module LoansHelper
   def back_to_loans
     session[:loans_path] || loans_path
   end
+
+  def health_status_message(loan)
```

Comment:
This method only needs a health_check object, not a full loan. Best to depend on the smallest amount of knowledge possible.